src/contexts/CompendiaContext.js

```diff
@@ -0,0 +1,24 @@
+import { createContext, useMemo, useState } from 'react'
+
+export const CompendiaContext = createContext({})
+
+export const CompendiaContextProvider = ({ children }) => {
```

Comment:
Why not instead of putting the `CompendiaContext.Provider` in _app you just use it normally and pass in the default values to the provider?

Reply:
I've move `initialCompendia`, `initialType`, and the `useEffect` block from `useCompendiaContext` to `CompendiaContextProvider`,  and cleaned up the initialization in the `pages/compendia/[type]` component 👍